package.json

```diff
@@ -28,8 +28,8 @@
     "govuk-frontend": "^4.7.0",
     "jquery": "3.7.1",
     "mini-css-extract-plugin": "^2.8.1",
-    "postcss": "^8.4.35",
-    "sass": "^1.72.0",
+    "postcss": "^8.4.38",
+    "sass": "^1.71.1",
```

Comment:
When you resolve the conflicts it looks as though sass has been downgraded. This is because sass was upgraded in a previous commit This commit should be edited to keep the higher version of sass. The same goes for turbo-rails, further up the file. With this, the subsequent commit, to re-run yarn, is unecessary.

Reply:
Thanks @jrmhaig. I think I am doing them in the wrong order! I need to make sure I check though.